src/crafting.rs

```diff
@@ -336,6 +339,7 @@ pub struct ProfitableItem {
     pub crafting_steps: Rational32,
     pub count: i32,
     pub profit: Rational32,
+    pub min_price: i32,
```

Comment:
It doesn't make a difference here, but generally it's been useful to keep things as `Rational32` until they need to be printed

Reply:
I wanted the `add_trading_post_sales_commission` function to be an exact inverse of the `subtract_trading_post_sales_commission` function, so the type signatures are inverted. Would you rather it be not quite an inverse?

Reply:
It's good like this